src/zenml/models/v2/core/run_metadata.py

```diff
@@ -81,14 +79,11 @@ class RunMetadataResponseBody(WorkspaceScopedResponseBody):
 class RunMetadataResponseMetadata(WorkspaceScopedResponseMetadata):
     """Response metadata for run metadata."""
 
-    pipeline_run_id: Optional[UUID] = Field(
-        title="The ID of the pipeline run that this metadata belongs to.",
-    )
-    step_run_id: Optional[UUID] = Field(
-        title="The ID of the step run that this metadata belongs to."
+    resource_id: UUID = Field(
```

Comment:
It might be nice to put the `resource_id` and the `resource_type` into the body of the response as they are essential to a `RunMetadataResponse`. WDYT?

Reply:
I disagree, in almost all cases you will access this models through the parent entity, e.g., `artifact.run_metadata` / ... in which case you already know what the resource type and ID are